src/main/kotlin/com/goldenraven/tatooine/Application.kt

```diff
@@ -64,6 +66,7 @@ fun Application.module(testing: Boolean = false) {
         authenticate("padawan-authenticated") {
             post("/sendcoins") {
                 val address: String = call.receiveText()
+                println(address)
```

Comment:
Please add a simple message so that the reader of the logs knows what is being printed! For example you could say:

```kotlin
println("Coins being sent to address $address")
```